app/views/feeds/_form_status.html.erb

```diff
@@ -5,11 +5,50 @@
   <div class="card-footer d-flex align-items-center">
     <% if feed.persisted? %>
       <p class="status-text flex-grow-1"><strong><%= t(".updated_at_hint") %></strong> <br><%= l(feed.updated_at) %></p>
-      <%= link_to t(".discard"), podcast_feed_path(podcast, feed), class: "btn btn-discard-changed invisible me-2", data: {action: "unsaved#discard", unsaved_target: "discard"} %>
-      <%= form.submit t(".update"), class: "btn btn-primary" %>
+
+      <% if policy(feed).update? %>
+        <%= link_to t(".discard"), podcast_feed_path(podcast, feed), class: "btn btn-discard-changed invisible me-2", data: {action: "unsaved#discard", unsaved_target: "discard"} %>
+
+        <div class="btn-group">
+          <% data = {disable_with: t(".updating"), upload_with: t(".uploading")} %>
+          <%= form.submit t(".update"), class: "btn btn-primary", data: data %>
+
+          <% if policy(feed).destroy? && !feed.default? && !feed.new_record? %>
```

Comment:
destroy button only allowed on persisted feeds that aren't the default. is this true? would the user ever need to be able to delete the default feed?

Reply:
You got it!  Deleting the default feed would probably cause "bad things" to happen.